library/ix-dev/community/rsyncd/ci/hostnet-multiple-values copy.yaml

```diff
@@ -0,0 +1,59 @@
+rsyncNetwork:
```

Comment:
You have a space in the file name..is that okay?
I would assume yes - it's just that it's not common to see that :D

Reply:
There isn't an issue, but good catch! fixed!